noir-projects/noir-contracts/contracts/token_with_refunds_contract/src/main.nr

```diff
@@ -186,13 +462,20 @@ contract TokenWithRefunds {
             fee_payer_randomness
         );
 
-        // 5. Set the public teardown function to `complete_refund(...)`. Public teardown is the only time when a public
+        // 5. Now we "manually" compute the slots and the slotted note hiding points
```

Comment:
Here we "insert" the notes to the balances Map. Due to the homomorphic properties of the points we can now do this in private! 🎉🥳🎊🍾🎈🎆🥂👏